src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -89,6 +89,51 @@
           </template>
         </ChiDataTableToolbar>
       </template>
+      <template #bulkActions>
+        <ChiDataTableBulkActions
```

Comment:
This pattern requires tracking of the number of the selected rows to provide the number back to `<ChiDataTableBulkActions />`. This adds an additional layer of functionality for the developers to take care of.
I suggest we avoid that.

`_bulkAction()` method of `DataTable.tsx` should return the `<ChiDataTableBulkActions selectedRows={NUMBER_OF_SELECTED_FIRST_LEVEL_ROWS}>
{this.$scopedSlots['bulkActions']({}) || null}
</ChiDataTableBulkActions>`.

I think we should achieve this:
```
<template #bulkActions>
  <template v-slot:start>
    BUTTONS HERE
  </template>
</template>
```

Instead of providing the whole `<ChiDataTableBulkActions>` and tracking the number of selected rows from outside.